bindings/csharp/CMakeLists.txt

```diff
@@ -8,7 +8,8 @@ INCLUDE_DIRECTORIES(
 	${CMAKE_SOURCE_DIR}/include
 	${CMAKE_CURRENT_SOURCE_DIR})
 
-SET(CMAKE_SWIG_FLAGS "")
+set(CMAKE_SWIG_FLAGS "")
+
```

Comment:
This can be removed from the changes, right?

Reply:
Removed in commit d74555b.